openstack/python/tar/slipstream_openstack/OpenStackClientCloud.py

```diff
@@ -71,7 +71,20 @@ def __init__(self, configHolder):
     def _initialization(self, user_info):
         util.printStep('Initialize the OpenStack connector.')
         self._thread_local.driver = self._get_driver(user_info)
-        self.flavors = self._thread_local.driver.list_sizes()
+
+        try:
+            self.flavors = self._thread_local.driver.list_sizes()
+        except InvalidCredsError as e:
+            raise Exceptions.ValidationException("Invalid Cloud credentials. "
+                                                 "Please check your Cloud username, password, project name and domain "
+                                                 "(if applicable) in your SlipStream user profile.")
+        except LibcloudError as e:
+            if e.message == 'Could not find specified endpoint':
+                raise Exceptions.ValidationException("Invalid Cloud configuration. "
+                                                     "Please ask your SlipStream administrator to check the region, "
+                                                     "service-type and service-name.")
```

Comment:
Is the connector instance name known here? If so, it would be handy to add it to the message. What do you think?